app/Http/Controllers/UserController.php

```diff
@@ -92,7 +92,7 @@ public function admin($userid){
 
     }
 
-    public function admin_store(Request $request, $userid){
+    public function admin_store(Request $request, $userid) {
```

Comment:
__[code_sniffer]__
* Method name "UserController::admin_store" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->